### PR TITLE
Fixed detection of a version of the AMI protocol

### PR DESCRIPTION
**Pull request by s-wave**

### CODE REVIEW

File: panoramisk/ami_protocol.py [hunk 1]
```diff
@@ -50,8 +50,11 @@ def data_received(self, data):
         # self.log.debug('data received: "%s"', data)
 
         if self.version is None:
-            if data.startswith('Asterisk Call Manager/'):
-                self.version = data.strip().split('/')[1]
+            if data.startswith(b'Asterisk Call Manager/'):
```
Comment (gawel) [inline panoramisk/ami_protocol.py]: I dont really understand why you're using bytes. data are already decode at https://github.com/ssash/panoramisk/blob/53d1ff82caf46e7fd3b188edce4cd7015ba047e2/panoramisk/ami_protocol.py#L41

Comment (ssash) [reply panoramisk/ami_protocol.py]: Oops! It's my mistake.
I'll correct this.

**Comment by gawel** [comment]
I don't really understand why you need this. But it shouldn't break anything. So LGTM

**Comment by s-wave** [comment]
The requirements for my application are such that it is necessary support connection to Asterisk-11 (AMI/1.3) and Astrisk-13 (AMI/2.10.4).
And I create AMI message dispatcher according to AMI version after successfull connection to Asterisk.

**Pull request merged**